tensorflow_serving/model_servers/server.cc

```diff
@@ -303,7 +303,8 @@ Status Server::BuildAndStart(const Options& server_options) {
   }
   builder.RegisterService(model_service_.get());
   builder.RegisterService(prediction_service_.get());
-  builder.SetMaxMessageSize(tensorflow::kint32max);
+  builder.SetMaxReceiveMessageSize(tensorflow::kint32max);
+  builder.SetMaxSendMessageSize(tensorflow::kint32max);
```

Comment:
why set send message size limits? we just need to set max receive, given that
current method in grpc just sets this limit for SetMaxMessageSize() method:

https://github.com/grpc/grpc/blob/deaa8e16b3a9224a8447018008255cd7096d2443/include/grpcpp/server_builder.h#L205